src/services/validation.service.js

```diff
@@ -4,6 +4,8 @@ import path from 'path'
 // eslint-disable-next-line max-len
 const ipv4 = /^\s*((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))\s*$/
 // eslint-disable-next-line max-len
+const adsNetId = /^\s*((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))\s*$/
+// eslint-disable-next-line max-len
```

Comment:
should we put "driver" dependant validation in this validation.service.js supposed to be more generic?

Reply:
For now I let it here : we should first discuss how to manage specific validation : do we put it in a 'local' validation service in the associated connector ?